tdp/cli/commands/deploy.py

```diff
@@ -82,7 +82,11 @@ def deploy(
             ),
             cluster_variables=cluster_variables,
             cluster_status=dao.get_cluster_status(),
-        ).run(planned_deployment, force_stale_update=force_stale_update)
+        ).run(
+            planned_deployment,
+            force_stale_update=force_stale_update,
+            ansible_inventory_file=str(run_directory) + "/inventory/inventory.ini",
```

Comment:
You can not hard code the inventory file to read. This is why we have all the boilerplate inside `InventoryReader` to let Ansible parse config files and load all inventory files. The downside is inventory files are read when `ansible` module is imported and inventory is read once.

Reply:
Yes you are right, I did all these because I did not see that there was a `refresh_inventory()` module for the InventoryManager. The code has been refactored and simplified.